applications/RANSApplication/custom_elements/convection_diffusion_reaction_stabilization_utilities.h

```diff
@@ -60,18 +61,15 @@ inline void CalculateStabilizationTau(
     const double DeltaTime,
     const double DynamicTau)
 {
-    unsigned int dim = rContravariantMetricTensor.size2();
-    const Vector& velocity = RansCalculationUtilities::GetVector(rVelocity, dim);
-    Vector temp(dim);
-    noalias(temp) = prod(rContravariantMetricTensor, velocity);
+    const array_1d<double, TDim> temp = prod(rContravariantMetricTensor, rVelocity);
```

Comment:
`temp` is only used in line 68, so it should be moved into that branch

Reply:
done :)